src/libopal.c

```diff
@@ -2392,6 +2392,45 @@ short print_asm_code(asm_cmd_e cmd_list[], FILE *dest_fp)
   /// Copy NASM footer file to dest_fp
 
   /// Create strings and their lengths
+  for (int i = 0; i < strs_len; i++)
+    {
+      fprintf (dest_fp, "msg%d: DB \"", i);
+      /// Read each string character
+      for (int j = 0; j < strlen(strs[i]); j++)
+        {
+           ///print ASCII values for newlines
+           if (strs[i][j] == '\\' && strs[i][j+1] == 'n')
+             {
+               fprintf (dest_fp, "\", 13, 10, \"");
+               j = j+ 1;
+             }
+
+           /// directly print all other characters
+           else
+             fprintf (dest_fp, "%c", strs[i][j]);
+        }
+
+      /// NULL terminate string
+      fprintf (dest_fp, ", NULL\n");
```

Comment:
You need to cap the other end of the string with a matching `"` that starts just before the `for` loop .. 

```
fprintf (dest_fp, "\", NULL\n");
```

Reply:
Sorry I thought I fixed that, I've been pasting this into a replit to test and apparently I only fixed it there.